libethashseal/genesis/transitionnetTest.cpp

```diff
@@ -56,6 +56,10 @@ R"E(
 		"0000000000000000000000000000000000000002": { "wei": "1", "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
 		"0000000000000000000000000000000000000003": { "wei": "1", "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
 		"0000000000000000000000000000000000000004": { "wei": "1", "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } }
+		"0000000000000000000000000000000000000005": { "wei": "1", "precompiled": { "name": "modexp" } },
```

Comment:
transitionnet test has the Metropolis block set to 
"metropolisForkBlock": "0x10"   (hex)